openage/convert/changelog.py

```diff
@@ -35,7 +35,7 @@
     {"graphics"},
     {"interface"},
     {"interface"},
-    {"gamedata"},
+    {"gamedata", "metadata"},
```

Comment:
There is no `gamedata` change, the available components are in the map above (`COMPONENTS`).